src/main/java/com/violeth/blockjs/blockjs/binds/Binds.java

```diff
@@ -81,6 +82,35 @@ public void removeBlockBreakEventListener(int id) {
         for (var blockBinds : onBlockBreak) {
             var removedBind = blockBinds.map.remove(id);
 
+            if (removedBind != null) {
+                return;
+            }
+        }
+    }
+    public int addBlockInteractEventListener(int x, int y, int z, V8Function callback) {
```

Comment:
Also is it at all possible to make it so that the callback function isn't ran over and over? What I mean is if I add a listener now and I hold down the right click on the block it runs the function over and over until I let go of the button

Reply:
I see, alright it's fine for now. I'll take a look at it later after I merge I think, see if I can't figure something out.